eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -32,10 +32,5 @@
     <UsagePattern IdentityGlob="System.Diagnostics.EventLog/*7.0.0*" />
     <UsagePattern IdentityGlob="System.Reflection.MetadataLoadContext/*7.0.0*" />
     <UsagePattern IdentityGlob="System.Security.Cryptography.ProtectedData/*7.0.0*" />
-    
-    <!-- msbuild is still on 8.0 versions of these so we have to pin in SDK main for now or tasks will fail -->
```

Comment:
Does this comment no longer apply? Did MSBuild switch to version 9.0 of these assemblies?

Reply:
I'm a bit confused by that comment.

I don't understand how these assemblies are related to msbuild since System.Text.Json, the ref that was related to these other dependencies per https://github.com/dotnet/sdk/pull/35048/files#r1330753409, is a [7.0 version in msbuild](https://github.com/dotnet/msbuild/blob/5fcddc790f4eeaf953a3d283e39751dd0e1f2992/eng/Version.Details.xml#L55). Second, these prebuilts were added as a workaround to an issue with the sdk tasks as part of https://github.com/dotnet/sdk/pull/35048. https://github.com/dotnet/sdk/pull/35048/files#r1330753409 suggests that the issue was related to the fact that System.Text.Json was on 8.0, but msbuild is on 7.0?

This PR is getting a clean build and my local copy of the VMR with these changes is also building successfully. Based on this, I'm assuming that this comment no longer applies, but as I pointed out above, I'm a bit confused as to how the comment was relevant in the first place.

@marcpopMSFT - What are your thoughts?

Reply:
If the full framework CI leg is passing then it's probably OK, for now at least.  When running under Full Framework MSBuild, there can be assembly loading failures if there isn't a binding redirect for the assembly and the assembly versions don't match exactly.  (We just encountered this when building 8.0.1, where we were trying to use version 8.0.1 of System.Text.Json, but Microsoft.Extensions.DependencyModel depended on version 8.0.0.  The resolution was to not update to the new version of System.Text.Json for the full Framework task builds.)